webapp/src/main/java/com/box/l10n/mojito/entity/ScheduledJobType.java

```diff
@@ -0,0 +1,45 @@
+package com.box.l10n.mojito.entity;
+
+import static org.hibernate.envers.RelationTargetAuditMode.NOT_AUDITED;
+
+import com.box.l10n.mojito.rest.View;
+import com.fasterxml.jackson.annotation.JsonView;
+import jakarta.persistence.Basic;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import org.hibernate.envers.Audited;
+
+@Entity
+@Table(name = "scheduled_job_type")
+@Audited(targetAuditMode = NOT_AUDITED)
+public class ScheduledJobType extends BaseEntity {
+  @Id private Long id;
+
+  @Basic(optional = false)
```

Comment:
Hold on. Why is this needed? The column is not not nullable, so should the config not reflect that. This Basic property to my understanding enforces that the property is not null locally but not at a db level (I might be wrong)

Reply:
I have updated the other columns that are not nullable to have this, I noticed it was on other entity objects so I decided to use it. Also it looks like you can generate SQL schemas off of entity mappings, not that we would need to use this but best practice to have annotations like this!